src/common/format/template.js

```diff
@@ -1,6 +1,53 @@
-const trim = (s) => s.replace(/^\s+|\s+$/g, '');
-const identity = (x) => x;
+const trim = (s) => s.trim();
 
+function safe(fn) {
+  return function wrapped(...args) {
```

Comment:
Just that I get things right: `wrapped` is a _"new"_ name for the thing you are returning here?

Reply:
Yeah, with "the thing" being a new function that runs the original function `fn` with the arguments it receives, but catches errors and turns them into strings. This makes sure that when someone is using the helper(-creators) incorrectly, the options page etc. don't break and you get a somewhat helpful output instead.

Reply:
I actually wrote this without a name for the returned function first, but our linting rules (sensibly) require us to provide a name.